wire-ios-mono.xcworkspace/contents.xcworkspacedata

```diff
@@ -109,7 +109,4 @@
    <FileRef
       location = "group:wire-ios-system/WireSystem.xcodeproj">
    </FileRef>
-   <FileRef
-      location = "group:WireAPI">
-   </FileRef>
```

Comment:
should not be there ?

Reply:
WireAPI is not merged yet, the reference was accidentally committed.